<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	background-color: rgb(0, 0, 0);
	color: rgb(191, 191, 191);
}



QLineEdit{
	border: solid;
	border-width: 1px;
	border-color: rgb(255,255,255);
}

QDoubleSpinBox{
	border: solid;
	border-width: 1px;
	border-color: rgb(255, 255, 255);
}

QPushButton {
	text-align: center;
	padding: 0px 00px;
	color: rgb(255, 255, 255);
	background-color: rgb(31, 31, 31); 
	border: solid;
	border-width: 2px; 
	border-radius: 10px; 
	border-color: rgb(127, 127, 127)
}
QPushButton:pressed {
	background-color: rgb(64, 64, 64); 
	border-style: inset
}</string>
  </property>
  <widget class="QRadioButton" name="punctLfRB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>320</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>LoadFlow puntuale</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="profLfRB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>320</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>LoadFlow nell'intero intervallo di tempo</string>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarCWgt">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>320</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QPushButton" name="calcPls">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Calcola</string>
   </property>
  </widget>
  <widget class="QWidget" name="datetimeWgt" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QLabel" name="timeSepLbl">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>0</y>
      <width>10</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="dateLE">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>90</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>21/12/2002</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="sepLbl">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>20</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="minuteDsb">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>30</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="singleStep">
     <double>15.000000000000000</double>
    </property>
    <property name="value">
     <double>30.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="hourDsb">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>30</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="maximum">
     <double>23.000000000000000</double>
    </property>
    <property name="value">
     <double>23.000000000000000</double>
    </property>
   </widget>
   <zorder>timeSepLbl</zorder>
   <zorder>sepLbl</zorder>
   <zorder>minuteDsb</zorder>
   <zorder>hourDsb</zorder>
   <zorder>dateLE</zorder>
  </widget>
  <zorder>punctLfRB</zorder>
  <zorder>profLfRB</zorder>
  <zorder>calcPls</zorder>
  <zorder>datetimeWgt</zorder>
  <zorder>calendarCWgt</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
